test/CadlRanchProjects/type/property/optionality/src/Generated/Models/UnionFloatLiteralPropertyProperty.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace _Type.Property.Optionality.Models
+{
+    /// <summary> The UnionFloatLiteralPropertyProperty. </summary>
+    public enum UnionFloatLiteralPropertyProperty
```

Comment:
this is weird, why we have this enum added? We should be able to generate enum before this change

Reply:
It's anonymous enum: https://github.com/Azure/cadl-ranch/blob/6c03a0c099dd910dfa318cc3c161be675577a85b/packages/cadl-ranch-specs/http/type/property/optionality/main.tsp#L175

> 1.25 | 2.375